hadoop-yarn-project/hadoop-yarn/bin/yarn

```diff
@@ -180,6 +180,9 @@ ${HADOOP_COMMON_HOME}/${HADOOP_COMMON_LIB_JARS_DIR}"
         HADOOP_HEAPSIZE_MAX="${YARN_ROUTER_HEAPSIZE}"
       fi
     ;;
+    routeradmin)
+      HADOOP_CLASSNAME='org.apache.hadoop.yarn.client.cli.RouterCLI'
```

Comment:
This PR adds a router admin CLI and a method to deregister methods.
Can we split it?

Reply:
Thank you very much for your help in reviewing the code, I agree with your suggestion, I will split this pr.